### ISSUE TITLE
Does it support Big Sur?

### ISSUE THREAD

**Issue by hayd1n** [zachs78/MacOS-XPS-9500-OpenCore#21]
Are there anyone tried running Big Sur on XPS 9500 with this EFI?

**Comment by daviddean-x** [comment]
yes

**Comment by daviddean-x** [comment]
take a look to my repo

**Comment by hayd1n** [comment]
> take a look to my repo

Is sleep working?

**Comment by daviddean-x** [comment]
No, till dell won't fix s3 deep sleep over an update, there's no way or someone would port the linux workaround to macos.
sleep is the only issue till yet.
You can disable sleep, and turn off your screen this working fine.
